src/terminal/adapter/FontBuffer.cpp

```diff
@@ -515,11 +515,11 @@ void FontBuffer::_fillUnusedCharacters()
     // with an error glyph (a reverse question mark). This includes every
     // character prior to the start char, or after the last char.
     const auto errorPattern = _generateErrorGlyph();
-    for (size_t ch = 0; ch < MAX_CHARS; ch++)
+    for (til::CoordType ch = 0; ch < MAX_CHARS; ch++)

```

Comment:
This doesn't seem to be a coordinate type either, but it is used in a _coord-expr_ later.